.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
I think this should be left up to one's own personal preferences, no reason to force autoformatting on save if one's not used to it.

```suggestion
```